src/components/generic/movieModal/modalContent.tsx

```diff
@@ -117,4 +121,12 @@ const ModalContent: FC<Props> = ({
   )
 }
 
-export default ModalContent
+const Category: FC<{title: string; children: React.ReactChild}> = ({
+  title,
+  children,
+}) => (
```

Comment:
```suggestion
const Category = ({
  title,
  children,
}: {title: string; children: React.ReactChild}) => (
```

I believe that you don't need `FC` if you are explicitly typing `children`.

Reply:
Done